Wikipedia/Code/NSUserDefaults+WMFApplicationDefaults.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+extension NSUserDefaults {
+    public func wmf_setReadingFontSize(fontSize: NSNumber) {
+        self.setObject(fontSize, forKey: "ReadingFontSize")
+        self.synchronize()
+    }
+    
+    public func wmf_readingFontSize() -> NSNumber {
+        if let fontSize = self.objectForKey("ReadingFontSize") as? NSNumber {
+            return fontSize
+        } else {
+            let preferredContentSizeCategory = UIApplication.sharedApplication().preferredContentSizeCategory
+            switch preferredContentSizeCategory {
+            case UIContentSizeCategoryExtraSmall:
+                return NSNumber(integer: 70)
```

Comment:
Maybe use constants for these `70, 85, 100`... etc sizes so we can DRY between this method and those in `- (NSArray<NSNumber *> *)fontSizeMultipliers`?